doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -5,21 +5,91 @@
 
 ## Table of Contents
 * [1.1 Introduction.](#1.1)
-* [1.2 Terminology.](#1.2)
-* [1.3 Approach](#1.3)
-* [1.4 Principles](#1.4)
-* [1.5 scope](#1.4)
-* [1.6 Vision](#1.4)
-* [1.7 Roadmap](#1.4)
+* [1.2 Terminology](#1.2)
+* [1.3 Principles](#1.3)
+  * [1.3.1 Cloud Native.](#1.3.1)
+* [1.4 Scope](#1.4)
+* [1.5 Vision](#1.5)
+* [1.6 Approach](#1.6)
+* [1.7 Roadmap](#1.7)
+
 
 <a name="1.1"></a>
 ## 1.1 Introduction
 
-This document (at this stage, not a RA, we're just using the template as a placeholder), discusses the role of Kubernetes in managing NFV software components (e.g. VNFCs). Application containers such as those managed by Kubernetes can be considered a Foundational PaaS capability<sup>1</sup> and so the following terms can all be thought of as being the topic of this document:
-- Kubernetes PaaS (Platform-as-a-Service)
-- CaaS (Containers-as-a-Service)
-- Kubernetes-based CaaS (Containers-as-a-Service)
-- Kubernetes-as-a-Service
+> to be written
+
+
+<a name="1.2"></a>
+### 1.2 Terminology
+
+|Term|Definition|Reference|
+|---|---|---|
+|Container Image|A container image is a lightweight, standalone, executable package of software that includes everything needed to run an application: code, runtime, system tools, system libraries and settings.|[Link](https://www.docker.com/resources/what-container)|
+|Container|A running instantiation of a container image.|-|
+|Pod|A Pod (as in a pod of whales or pea pod) is a group of one or more containers (such as Docker containers), with shared storage/network, and a specification for how to run the containers. A Pod’s contents are always co-located and co-scheduled, and run in a shared context.|[Link](https://kubernetes.io/docs/concepts/workloads/pods/pod/)|
```

Comment:
I think we should add that pod is the atomic unit of LCM in Kubernetes and co-located in the same host.

Reply:
To address his, we could add the half sentence following the linked definition, too:
"A Pod models an application-specific “logical host” - it contains one or more application containers which are relatively tightly coupled."
An alternative would be to quote the definition in the glossary:
https://kubernetes.io/ja/docs/reference/glossary/?fundamental=true#term-pod

Reply:
> 
> 
> To address his, we could add the half sentence following the linked definition, too:
> "A Pod models an application-specific “logical host” - it contains one or more application containers which are relatively tightly coupled."
> An alternative would be to quote the definition in the glossary:
> https://kubernetes.io/ja/docs/reference/glossary/?fundamental=true#term-pod

I agree, but I would rather link the [English documentation page](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-pod) than the Japanese...

Reply:
we need to add, "A pod is the smallest unit that can be created and managed by kubernetes." We may want to clearly state that a pod is a logical deployment unit.

Reply:
I've used the Kubernetes glossary.  Will resolve this conversation.